### ISSUE TITLE
Update footer datahub stats to use number of site users

### ISSUE THREAD

**Issue by ChasNelson1990** [fjelltopp/ckanext-googleanalytics4#3]
### Problem

In our footers we show the data hub stats - great! However, as you can see in this screenshot, it's the number of _regsitered_ users.
![image](https://github.com/user-attachments/assets/7c2e8f87-f29d-44dc-afc6-36c50bc7a243)

Most of our clients care more about the number of site users, e.g. non-logged in viewers.

### Solution

Can we, in this extension, override https://github.com/fjelltopp/ckanext-fjelltopp-theme/blob/e077a1701ad24c4fa92febd9e8c77f71d67d5838/ckanext/fjelltopp_theme/helpers.py#L42 with a new helper that uses Google Analytics and returns the number of unique visitors to the site?

**Comment by ntwalibas** [comment]
@ChasNelson1990 Yes, that is possible by using the `activeUsers` metric. Please see https://developers.google.com/analytics/devguides/reporting/data/v1/api-schema, quoted below:
> `activeUsers`: The number of distinct users who visited your site or app.

I will just have to implement it in the existing PR.
